index.html

```diff
@@ -64,7 +64,7 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
 
```

Comment:
### Your Button
![image](https://user-images.githubusercontent.com/93039359/195178060-bbe445a3-f661-4259-ad5d-81d04d8406f3.png)

Your button doesn't have a 4.5:1 color contrast ratio. Kindly make it to follow the requrements and to make your portfolio accessible to color blind users :blush:

Reply:
Border and font color were updated. Now, the button is darker.